source/board/stm32h743ii.c

```diff
@@ -29,7 +29,7 @@ static void prerun_board_config(void) {
 
 const board_info_t g_board_info = {
     .info_version = kBoardInfoVersion,
-    .board_id = "0700",
+    .board_id = "0000",
```

Comment:
I'm curious where the 0700 came from, and if using 0000 will cause any issues (i.e. pyocd thinks it is some other board?)

from a quick search I see lots of numbers used, and 0000 sometimes used.  Not immediately obvious when "0000" is the right choice.  

source/board/frdmkl26z.c:    .board_id = "0260",
source/board/frdmkl27z.c:    .board_id = "0261",
source/board/frdmkl28z.c:    .board_id = "0205",
source/board/frdmkl43z.c:    .board_id = "0262",
source/board/frdmkl46z.c:    .board_id = "0220",
source/board/frdmkl82z.c:    .board_id = "0218",
source/board/frdmkw24d.c:    .board_id = "0250",
source/board/frdmkw41z.c:    .board_id = "0201",
source/board/gr-lychee.c:    .board_id = "5501",
source/board/gr-peach.c:    .board_id = "5500",
source/board/hani_iot.c:    .board_id = "0360",
source/board/hexiwear.c:    .board_id = "0214",
source/board/hrm1017.c:    .board_id = "1017",
source/board/hvpke18f.c:    .board_id = "0216",
source/board/k20dx_bl.c:    .board_id = "0000",
source/board/k26f_bl.c:    .board_id = "0000",
source/board/kl26z_bl.c:    .board_id = "0000",
source/board/kl27z_bl.c:    .board_id = "0000",
source/board/lpc4088dm.c:    .board_id = "1062",
source/board/lpc4088qsb.c:    .board_id = "1060",
source/board/lpc4322_bl.c:    .board_id = "0000",
source/board/lpc54114xpresso.c:    .board_id = "1054",

Reply:
Maybe we can add our custom board id into PyOCD, but it will take some time.
Since current version can not worked with PyOCD, we can use this commit as a temporary workaround.